ci/jjb/jobs/pulp3_PR_pulp-smash.yaml

```diff
@@ -0,0 +1,256 @@
+- job:
+    name: 'pulp-smash-pulp3-pr'
+    defaults: ci-workflow-runtest
+    project-type: matrix
+    node: 'master'
+    axes:
+      - axis:
+          type: label-expression
+          name: node-type
+          values:
+            - f27-pulp3-np
+
+    parameters:
+        - string:
+            name: sha1
+            default: ''
+            description: |
```

Comment:
Coud `>` be used instead of `|`? The pipe causes line breaks in the following text to be preserved. In contrast, the carat causes line breaks to be converted into spaces. Given that this is prose that's going to be rendered in UIs (e.g. HTML for web browsers), I don't see any reason to preserve line breaks.